src/Authorization/DefaultAuthorizationListener.php

```diff
@@ -21,20 +21,11 @@ class DefaultAuthorizationListener
     protected $authorization;
 
     /**
-     * Array of controller_service_name/identifier_name pairs
-     *
-     * @var array
-     */
-    protected $restControllers;
-
-    /**
-     * @param Acl $acl
-     * @param array $restControllers
+     * @param AuthorizationInterface $authorization
      */
-    public function __construct(AuthorizationInterface $authorization, array $restControllers = array())
+    public function __construct(AuthorizationInterface $authorization)
     {
         $this->authorization   = $authorization;
-        $this->restControllers = $restControllers;
```

Comment:
This isn't technically a BC break, because:

- Removal of arguments from a signature will not invalidate an extension.
- The property is never used.